package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "18": "^0.0.0"
+  }
```

Comment:
Not sure what this is, but I don't think there are supposed to be any `package-lock.json` or `package.json` files in the top level folder.